tests/setup.ts

```diff
@@ -1,6 +1,18 @@
+import dotenv from 'dotenv'
+import fs from 'fs'
+import path from 'path'
 import { Config } from '../src/'
 import { consoleLogger } from '../src/types/logging'
 
+let envPath = path.resolve(path.join(__dirname, '..', '.env'))
```

Comment:
We can do this in the `scripts` in package.json instead if we want, although not the switch from example env, but if we encourage use of algokit bootstrap all and the right .env.template name then it would work?